source/android/adaptivecards/src/main/java/io/adaptivecards/renderer/inputhandler/NumberInputHandler.java

```diff
@@ -50,8 +50,15 @@ public boolean isValidOnSpecifics(String numberInputValue)
         }
         catch (Exception ex)
         {
-            // Something failed, don't consider valid
-            return false;
+            // Parsing failed, consider it invalid if it was not required otherwise it's invalid
+            if (numberInputValue.isEmpty() && !numberInput.GetIsRequired())
```

Comment:
Does this need to be in the catch? Perhaps it makes more sense to move it above the try/catch